csharp/src/Google.Protobuf.Conformance/Conformance.cs

```diff
@@ -1,1590 +0,0 @@
-// <auto-generated>
```

Comment:
I see the Program.cs and ConformanceCSharp.csproj have been move to conformance/ConformanceCSharp, which is slightly less easy to work with - but more importantly, deleting the generated code makes it harder to work with too. For everything else, e.g. the address book, test protos etc we have the generated code checked in, so that it's ready to clone and build immediately. It's fine for the conformance test makefiles etc to regenerate, but I'd prefer the repo to be "immediately buildable out of the box" for C#.

Reply:
Ah yea that's reasonable.  I'll see if I can revert the move/deletions.  Long-term, it would be nice to configure the VS build to automatically generate the proto code.  However, we definitely have cases where we expect to be checking in generated code indefinitely (e.g. CMake build files), so we can just add staleness tests to make sure they don't get out of sync